### DISCUSSION TITLE
What is the largest shark species?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#489] [Galaxy Trivia] [answered]
*How long can they grow?*

**Comment by mayankjaiswal1709** [comment, marked as answer]
The largest shark species is the whale shark (Rhincodon typus), which can grow up to 40 feet (12.2 meters) in length and weigh up to 20 tons (18 metric tonnes). Whale sharks are filter feeders, which means they feed on plankton and small fish by swimming with their mouth open and filtering the water through their gills. Despite their massive size, they are generally considered to be gentle and pose no significant threat to humans.

**Reply by ParanoidUser** [reply]
Bingo! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/mayankjaiswal1709?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉

**Reply by mayankjaiswal1709** [reply]
ohky i will

**Comment by thuongtruong1009** [comment]
Sharks come in all sizes. The largest is the [whale shark](https://ocean.si.edu/ocean-photos/tagging-along-whale-sharks), which has been known to get as large as 18 meters (60 feet). The study revealed that male whale sharks can live as long as 130 years and may grow as large as 61.7 feet on average
